packages/eslint-config-globis/index.js

```diff
@@ -1,3 +1,13 @@
 module.exports = {
   extends: ['airbnb', 'airbnb/hooks', './shared.js'],
+  rules: {
+    'import/order': ['error', { 'newlines-between': 'always' }],
+    'import/prefer-default-export': 0,
+    // FixMe https://github.com/yannickcr/eslint-plugin-react/issues/1846
+    'react/button-has-type': 'off',
+    'react/jsx-filename-extension': [1, { extensions: ['.tsx'] }],
+    'react/jsx-key': ['error', { checkFragmentShorthand: true }],
+    'react/jsx-props-no-spreading': 0,
+    'react/prop-types': 0,
+  },
```

Comment:
![badge](https://img.shields.io/badge/review-imo-ff69b4.svg)
To simplify, I prefer these rules are defined to another one.

Reply:
I think it is a prerequisite configuration that uses React.

e.g) 
- https://github.com/globis-org/frontend-standard/blob/master/packages/eslint-config-globis/shared.js#L4
- https://github.com/globis-org/frontend-standard/blob/master/packages/eslint-config-globis/index.js#L2

Do we want to separate React configurations from the settings above?

Reply:
hmmm... 
We should clarify each file duty, to have same context. 🤓
It seem strange a little for me,  to `base.js` extends shared.js.

Reply:
> We should clarify each file duty, to have same context

+1